src/lib/isMobilePhone.js

```diff
@@ -133,6 +133,7 @@ const phones = {
   'zh-CN': /^((\+|00)86)?(1[3-9]|9[28])\d{9}$/,
   'zh-TW': /^(\+?886\-?|0)?9\d{8}$/,
   'dz-BT': /^(\+?975|0)?(17|16|77|02)\d{6}$/,
+  'en-BU': /^(\+?257|0)?\d{8}$/,
```

Comment:
Looks like landline numbers are passing with your regex, can you update it to allow only mobile phone numbers. (You can find the list of carrier prefixes here: https://en.wikipedia.org/wiki/Telephone_numbers_in_Burundi#List_of_mobile_codes_in_Burundi)